src/recipes/mulled-wine.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "<Mulled Wine>",
```

Comment:
```suggestion
    "name": "Mulled Wine",
```
You can remove all the less-than/greater-than symbols out. They are just placeholders.

Thanks!

Reply:
All edits have been made - thanks very much.

Reply:
Please let me know if I need to do anything further on my end.  Thanks!!